cms/apps/media/templates/admin/media/includes/file_used_on.html

```diff
@@ -0,0 +1,21 @@
+<fieldset class="module aligned with-legend ">
+  <h2 class="legend">Used on</h2>
+
+  <input style="display:none;" type="text" name="changed_image" maxlength="200" id="id_changed_image">
```

Comment:
What does this do?

Reply:
Nothing. Forgot to remove it from the code I took from the image editor :p.